pkg/ingester/ingester.go

```diff
@@ -803,9 +803,9 @@ func (i *Ingester) Watch(in *grpc_health_v1.HealthCheckRequest, stream grpc_heal
 	return status.Error(codes.Unimplemented, "Watching is not supported")
 }
 
-// ReadinessHandler is used to indicate to k8s when the ingesters are ready for
-// the addition removal of another ingester. Returns 204 when the ingester is
-// ready, 500 otherwise.
+// CheckReady is the readiness handler is used to indicate to k8s when the ingesters
+// are ready for the addition removal of another ingester. Returns 204 when the
+// ingester is ready, 500 otherwise.
```

Comment:
```suggestion
// are ready for the addition removal of another ingester.
```

HTTP status codes are not returned from here anymore. (Fixing my code change via your PR :))